webapp/app.py

```diff
@@ -50,21 +51,7 @@ def pluralize(count):
 
     @app.context_processor
     def inject_external_urls():
-        return dict(
-            external_urls={
-                "askubuntu": "https://askubuntu.com/questions/tagged/juju",
-                "charmstore": "https://api.jujucharms.com/charmstore/v5/",
-                "discourse": "https://discourse.jujucharms.com/",
-                "gettingStarted": (
-                    "https://docs.jujucharms.com/2.5/en/getting-started"
-                ),
-                "docs": "https://docs.jujucharms.com/",
-                "gui": "https://jujucharms.com/new/",
-                "issues": (
-                    "https://github.com/canonical-websites/jaas.ai/issues"
-                ),
-            }
-        )
+        return dict(external_urls=external_urls)
```

Comment:
It's easier to debug if this list remains here (less files/loops to find the code). Your call.

Reply:
I had to move it so that I could import it into another file which was also imported by this file, so there was an import loop.